stencil-library/src/components/business-form/business-form.tsx

```diff
@@ -22,33 +22,39 @@ export class BusinessForm {
   }
 
   sendData() {
-    console.log('sendData')
+    // send data to server
   }
 
   validateAndSubmit(event: any) {
     event.preventDefault();
-    this.formController.validateAndSubmit(this.sendData)
+    this.formController.validateAndSubmit(this.sendData);
   }
 
   render() {
     return (
       <Host exportparts="label,input,input-invalid">
         <h1>Business Information</h1>
-        <form onSubmit={(event) => this.validateAndSubmit(event)}>
-          <div class="row gy-3">
+        <form onSubmit={event => this.validateAndSubmit(event)} class="container">
```

Comment:
This change (`class="container"`) was in another PR and we decided not to include it. Should it be included here?

Reply:
Oh, no, it was just bad merging on my end.